### PR TITLE
Clean/afterbootcamp

### PR DESCRIPTION
**Pull request by matboivin**
Hi all!
First update after our last bootcamp session. Most issues have been closed. The ones that are left will be taken care of later.

### CODE REVIEW

File: day03/ex03/ex03.md [hunk 1]
```diff
@@ -12,37 +12,37 @@ For this exercise, the authorized functions and operators are specified for each
...
 
 `Bonus`: add an argument to your method to let the user choose the number of thresholds.  
-Authorized function : .vectorize, (.arange?)  
+Authorized functions: .vectorize, .arange
 Authorized operator: None  
 
 * `to_grayscale(array, filter)` : Takes a NumPy array of an image as an argument and returns an array in grayscale. The method takes another argument to select between two possible grayscale filters. Each filter has specific authorized functions and operators.  
 	* 'mean' or 'm' :  Takes a NumPy array of an image as an argument and returns an array in grayscale created from the mean of the RBG channels.  
-Authorized function : .sum, .shape, reshape, broadcast_to, (as_type?)  
+Authorized functions: .sum, .shape, reshape, broadcast_to, as_type
```
Comment (fxbabin) [inline day03/ex03/ex03.md]: missing dots in front of reshape, broadcast_to and as_type

File: day03/ex03/ex03.md [hunk 2]
```diff
@@ -12,37 +12,37 @@ For this exercise, the authorized functions and operators are specified for each
...
 
 * `to_red(array)` : Takes a NumPy array of an image as an argument and returns an array with a red filter.  
-Authorized function : green, blue  
+Authorized functions : green, blue  
 Authorized operator: -, +  
 
 * `celluloid(array)` : Takes a NumPy array of an image as an argument, and returns an array with a celluloid shade filter.
 The celluloid filter must display at least four thresholds of shades. Be careful! You are not asked to apply black contour on the object here (you will have to, but later...), you only have to work on the shades of your images.  
-Authorized function: arange, linspace  
+Authorized functions: .arange, linspace
```
Comment (fxbabin) [inline day03/ex03/ex03.md]: missing dot in front of linspace

**Comment by matboivin** [comment]
> need to add dots for allowed functions . May be included in the next merge

Thanks a lot! I created an issue for the next merge: #74

**Comment by matboivin** [comment]
Hi! I close this PR to add more changes in the next one.

**Pull request closed without merging**